src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFPicture.java

```diff
@@ -77,6 +77,20 @@ public XWPFPictureData getPictureData() {
         }
         return null;
     }
+   
+    /**
+     * Returns the width of the picture (in EMU). 
+     */
+    public long getWidth() {
+        return ctPic.getSpPr().getXfrm().getExt().getCx();
+    }
+   
+    /**
+     * Returns the width of the picture (in EMU). 
```

Comment:
`width` was copied from above.

Most of the other API use `Units.toPoints()` to convert out of EMU to more human understandable points.